testsuite/MDAnalysisTests/core/test_requires.py

```diff
@@ -59,9 +56,9 @@ def animal_print(ag):
         except NoDataError as e:
             message = e.args[0]
             # Test function name gets returned (for debug)
-            assert_('animal_print' in message)
-            assert_('cats' in message)
-            assert_('dogs' in message)
-            assert_('frogs' in message)
+            assert 'animal_print' in message
+            assert 'cats' in message
+            assert 'dogs' in message
+            assert 'frogs' in message
         else:
-            raise AssertionError("Should raise NoDataError")
+            raise pytest.fail(msg="Should raise NoDataError")
```

Comment:
No, [pytest.fail](https://docs.pytest.org/en/latest/builtin.html#_pytest.outcomes.fail) is a function and [should just be called](https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions):
```python
pytest.fail(msg="Should raise NoDataError")
```
(See also [real world examples](http://www.programcreek.com/python/example/58624/pytest.fail)).